PyInstaller/utils/hooks/gi.py

```diff
@@ -140,9 +141,10 @@ def gir_library_path_fix(path):
                          'package.', gir_file)
             return None
 
-        with open(gir_file, 'r') as f:
+        with open_file(gir_file, text_read_mode, encoding='utf-8') as f:
```

Comment:
How can we be sure this is utf-8 encoded? If you have some source for this assumption, please state this in a comment.

Reply:
As before, let's specify and encoding, instead of relying on a poorly-documented "default".

Reply:
Looks like git files are XML https://developer.gnome.org/gi/stable/gi-gir-reference.html. Thus strictly speaking we should use an xml parser which cares about the correct encoding. But since utf-8 is the XML default encoding, we can life with this. Please add a respective commend.

Reply:
Done, thanks.